grails-app/views/requisition/_showRequisitionItem.gsp

```diff
@@ -165,7 +197,44 @@
         <g:else>
             ${requisitionItem?.calculateQuantityRemaining()?:0}
         </g:else>
-
     </td>
+    <g:if test="${requestTab}">
+        <td class="middle center">
+            <g:if test="${requisitionItem?.isSubstituted()}">
+                <div>
+                    ${requisitionItem?.substitutionItem?.quantityIssued?:0}
+                </div>
+            </g:if>
+            <g:elseif test="${requisitionItem?.isCanceled()}">
+                <div class="canceled">
+                    ${requisitionItem?.quantityIssued?:0}
+                </div>
+            </g:elseif>
+            <g:elseif test="${requisitionItem?.isChanged()}">
+                <div>
+                    ${requisitionItem?.modificationItem?.quantityIssued?:0}
+                </div>
+            </g:elseif>
+            <g:else>
+                ${requisitionItem?.quantityIssued?:0}
+            </g:else>
+        </td>
+    </g:if>
+    <g:if test="${requestTab}">
+        <td class="middle center">
+            <g:set var="pickReasonCode" value="${requisitionItem?.picklistItems?.find { it.reasonCode } ?:
```

Comment:
In the future, add complicated logic like this as a method to RequisitionItem (i.e. getPickReasonCode()).